docs/installation.rst

```diff
@@ -273,11 +273,27 @@ Many of Pillow's features require external libraries:
 
 .. tab:: Windows using MSYS2/MinGW
 
-    To build Pillow using MSYS2, make sure you run the **MSYS2 MinGW 32-bit** or
-    **MSYS2 MinGW 64-bit** console, *not* **MSYS2** directly.
-
-    The following instructions target the 64-bit build, for 32-bit
-    replace all occurrences of ``mingw-w64-x86_64-`` with ``mingw-w64-i686-``.
+    The following instructions are for the **MSYS2 MINGW64** system.
```

Comment:
[MINGW64 is no longer recommended.](https://www.msys2.org/news/#2022-10-29-changing-the-default-environment-from-mingw64-to-ucrt64) Please update everything accordingly.